packages/tc-ui/src/primitives/simple-relationship/server.jsx

```diff
@@ -0,0 +1,75 @@
+const React = require('react');
+const { getType } = require('@financial-times/tc-schema-sdk');
+const { WrappedEditComponent } = require('../../lib/components/input-wrapper');
+const { RelationshipPicker } = require('../relationship-picker');
+const { SelectedRelationship } = require('./lib/selected-relationship');
+
+const RelationshipPickerContainer = props => (
+	<div>
+		<RelationshipPicker
+			{...props}
+			SelectedRelationship={SelectedRelationship}
+			componentName="simple-relationship-picker"
+		/>
+	</div>
```

Comment:
No need for additional `<div>`

Reply:
Needed for styling because flexbox I think. I can leave a comment